src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started work");

```

Comment:
The message should indicate what type of `Machine` is working right now `Truck`, `Bulldozer` or `Excavator`. Consider rephrasing to include the type of the machine, for example: 'Bulldozer is doing its work.'